packages/sw-lib/src/index.js

```diff
@@ -27,53 +26,37 @@ if (!assert.isSWEnv()) {
 }
 
 /**
- * The sw-lib module is a high-level library that makes it easier to
- * configure routes with caching strategies as well as manage precaching
+ * A high-level library that makes it easier to configure routes with
+ * caching strategies as well as manage precaching
  * of assets during the install step of a service worker.
  *
- * @example
+ * @example <caption>Adding the Library to your project.</caption>
  * importScripts('/<Path to Module>/build/sw-lib.min.js');
  *
- * // cacheRevisionedAssets() can take an array of strings with
- * // the revision details in the url.
- * goog.swlib.cacheRevisionedAssets([
- *     '/styles/main.1234.css',
- *     '/images/logo.abcd.jpg'
- * ]);
- *
- * // Or it can accept objects with the URL and revision data
- * // kept seperate.
- * goog.swlib.cacheRevisionedAssets([
- *     {
- *       url: '/index.html',
- *       revision: '1234'
- *     },
- *     {
- *       url: '/about.html',
- *       revision: 'abcd'
- *     }
- * ]);
+ * console.log(self.goog.swlib);
  *
- * // If you have assets that aren't revisioned, you can cache them
- * // during the installation of you service worker using warmRuntimeCache()
- * goog.swlib.warmRuntimeCache([
- *     '/scripts/main.js',
- *     '/images/default-avater.png'
- * ]);
+ * @example <caption>Cache some assets and register routes.</caption>
```

Comment:
Use participle phrases for captions. >> "Caching assets and registering routes."